src/core/tests/minhash.rs

```diff
@@ -136,6 +136,18 @@ fn dayhoff() {
     assert_eq!(b.size(), 2);
 }
 
+#[test]
```

Comment:
Is this addition in this PR intentional?

Reply:
I had to trigger a rust change, so why not write a test? =)